prow/scripts/build-compass-development-artifacts.sh

```diff
@@ -8,25 +8,22 @@
 
 set -e
 
-discoverUnsetVar=false
+readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck disable=SC1090
+source "${SCRIPT_DIR}/library.sh"
+# shellcheck disable=SC1090
```

Comment:
I would rather use there `# shellcheck source=prow/scripts/lib/utils.sh` instead of surpassing the shellcheck test. You can use relative path from test-infra repo.

Reply:
shellcheck line changed to `# shellcheck source=prow/scripts/lib/utils.sh`